src/MetaMorpho.sol

```diff
@@ -738,19 +771,17 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             uint256 toWithdraw = UtilsLib.min(_withdrawable(marketParams, id), remaining);
 
             if (toWithdraw > 0) {
-                // Using try/catch to skip markets that revert.
-                try MORPHO.withdraw(marketParams, toWithdraw, 0, address(this), address(this)) {
-                    remaining -= toWithdraw;
-                } catch {}
+                MORPHO.withdraw(marketParams, toWithdraw, 0, address(this), address(this));
+                remaining -= toWithdraw;
```

Comment:
We should maybe keep the try/catch here as broken markets can't be removed from the withdraw queue, and broken markets might not always be at the end of the withdraw queue

Reply:
Yes they can since we do a try/catch now!

Reply:
Apparently the try/catch is removed here